Source/WebCore/platform/graphics/Damage.h

```diff
@@ -142,7 +142,7 @@ class Damage {
         m_mode = Mode::Full;
         m_rects.clear();
         m_shouldUnite = false;
-        initialize();
+        initialize(0);
```

Comment:
I would suggest two named constants: `NoRectangles` (0) and `DefaultMax`  (ideally a better name, though) (32)

Reply:
all the more reason to name them, then :-)